### DISCUSSION TITLE
Option to list issues I've been involved

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [community/community#38386] [Projects and Issues]
Hi.

It always frustrated me on desktop and it still does in the mobile app.

I often need to find issues I commented and I only can by providing the search : `is:issue involves:my-username`

![Screenshot_20221105-232351_GitHub.png](https://user-images.githubusercontent.com/22337329/200143768-fce9ba69-126f-4baf-b41d-fc8bc446cc57.png)

The mobile app allows to create shortcuts for the home screen of the app. I'm stuck with the providing options. I can not create one that list all issues I've been involved.

**Comment by ossanna16** [comment]
Hi there @freMea and welcome to our community! Thank you for the feedback!

**Comment by Rochet2** [comment]
Finding PRs or issues you have commented on is quite difficult, especially on very active projects.
Would be very nice to get a feature to find any issues and PRs you have commented on. 👍
